cross/build-rootfs.sh

```diff
@@ -2,44 +2,82 @@
 
 usage()
 {
-    echo "Usage: $0 [BuildArch]"
-    echo "BuildArch can be: arm, arm64"
+    echo "Usage: $0 [BuildArch] [UbuntuCodeName]"
+    echo "BuildArch can be: arm, arm-softfp, arm64"
+    echo "UbuntuCodeName - optional, Code name for Ubuntu, can be: trusty(default), vivid, wily. If BuildArch is arm-softfp, UbuntuCodeName is ignored."
 
     exit 1
 }
 
 __CrossDir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 __InitialDir=$PWD
+
+# base development support
+__UbuntuPackages="build-essential"
+
+# symlinks fixer
+__UbuntuPackages+=" symlinks"
+
+# CoreCLR and CoreFX dependencies
+__UbuntuPackages+=" gettext"
+__UbuntuPackages+=" libcurl4-openssl-dev"
+__UbuntuPackages+=" libicu-dev"
+__UbuntuPackages+=" libkrb5-dev"
+__UbuntuPackages+=" liblttng-ust-dev"
+__UbuntuPackages+=" libssl-dev"
+__UbuntuPackages+=" libunwind8-dev"
+__UbuntuPackages+=" zlib1g-dev"
+
+if [ -z "$LLVM_ARM_HOME" ]; then
+    __LLDB_Package="lldb-3.6-dev"
+fi
```

Comment:
Currently yes. But no if I change
```
__UbuntuPackages+=" ${__LLDB_Package}"
```
below to
```
__UbuntuPackages+=" ${__LLDB_Package:-}"
```
It will automatically handle 'else' case.